packages/babel-preset-expo/src/__tests__/platform-shaking.test.ts

```diff
@@ -0,0 +1,110 @@
+import * as babel from '@babel/core';
+
+import preset from '..';
+
+function getCaller(props: Record<string, string>): babel.TransformCaller {
+  return props as unknown as babel.TransformCaller;
+}
+
+const DEFAULT_OPTS = {
+  babelrc: false,
+  presets: [[preset]],
+  plugins: [
+    // Fold constants to emulate Metro
+    require('metro-transform-plugins/src/constant-folding-plugin.js'),
+  ],
+  sourceMaps: true,
+  filename: 'unknown',
+  configFile: false,
+  compact: true,
+  comments: false,
+  retainLines: false,
+};
+
+function stripReactNativeImport(code: string) {
+  return code
+    .replace(
+      'var _interopRequireDefault=require("@babel/runtime/helpers/interopRequireDefault");var _Platform=_interopRequireDefault(require("react-native-web/dist/exports/Platform"));',
+      ''
+    )
+    .replace('var _reactNative=require("react-native");', '');
+}
+
+it(`removes Platform module usage on web`, () => {
+  const options = {
+    ...DEFAULT_OPTS,
+    caller: getCaller({ name: 'metro', engine: 'hermes', platform: 'web' }),
+  };
+
+  expect(
+    babel.transform(
+      `Platform.select({ ios: () => console.log('ios'), web: () => console.log('web'), android: () => console.log('android'), })`,
+      options
+    )!.code
+  ).toEqual(`(function(){return console.log('web');});`);
```

Comment:
Any reason to keep this code when the code right below tests a superset of Platform.select functionality?